cpp/models/ide_secir/model.h

```diff
@@ -118,18 +122,30 @@ class Model
     void compute_susceptibles(ScalarType dt);
 
     /**
-     * @brief Computes size of a flow.
+     * @brief Computes size of a flow for the current last time value in m_transitions.
      * 
      * Computes size of one flow from #InfectionTransition, specified in idx_InfectionTransitions, for the current 
-     * last time value in m_transitions.
+     * last time value in m_transitions. 
+     * The function compute_flow with one parameter more is called with the specification of current_time_index as 
+     * the last index of m_transitions.
+     * See also compute_flow for a description of the parameters.
```

Comment:
I would change the order of the compute_flow functions. I think it is easier to understand that way.